ui/app/AppLayouts/Profile/stores/PrivacyStore.qml

```diff
@@ -44,4 +44,12 @@ QtObject {
     function mnemonicWasShown() {
         root.privacyModule.mnemonicWasShown()
     }
+
+    readonly property bool thirdpartyServicesEnabled: appSettings.thirdpartyServicesEnabled
```

Comment:
It would be nice if we access to the `appSettings` context property via an instance here

Reply:
Seeing now this property in here, I think that there is a bit of messed up nomenclature with this other property from the comment here?  https://github.com/status-im/status-desktop/pull/18750#discussion_r2324707630

Reply:
@noeliaSD yes there are two properties -

1. privacyModeFeatureEnabled -> This is one is about we want this feature to be enabled altogether letting user turn OFF or ON privacy mode. Without it the option will not appear Onboarding or in Privacy Settings
2. thirdpartyServicesEnabled -> This holds ion the state has turned on or off third party services currently